app/assets/stylesheets/navigation.scss

```diff
@@ -0,0 +1,41 @@
+/************** NAVIGATION *******************************/
+
+#nav-container {
+  border-bottom: 1px solid #d9d9d9;
+  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown {
+  border: 1px solid #d9d9d9;
+  border-top: none;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.dropdown li > a {
+  border-bottom: 1px solid #d9d9d9;
```

Comment:
Color literals like `#d9d9d9` should only be used in variable declarations; they should be referred to via variable everywhere else.